aiven/client/cli.py

```diff
@@ -40,14 +42,14 @@
 EOL_ADVANCE_WARNING_TIME = timedelta(weeks=26)  # Give 6 months advance notice for EOL services
 
 
-def convert_str_to_value(schema, str_value):
-    if "string" in schema["type"]:
+def convert_str_to_value(schema: Mapping[str, Any], str_value: Optional[str]) -> Any:
+    if "string" in schema["type"] and str_value:
```

Comment:
if it's empty string, a `KeyError` or `ValueError` exception will be raised, and as I checked in the places we're using this function, we do catch the exception. And perhaps empty string is acceptable when schema type is string, not entirely sure tho.

Reply:
I mean the new logic is not the same as before. For example if `schema["type"]` is `string` and `str_value = ""` previously this function will return `""`. With the new changes it will raise `argx.UseError()` in the same case. Without checking all the client code it feels wrong. If this change is intended then it's OK.

In any case I think it's better to extract `and str_value` in the way I proposed in the first comment here.